src/index.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App/index';
+import App from './pages/Application/index';
```

Comment:
```suggestion
import App from './pages/application';
```
Only the name of the components folders should be uppercase, here `application` is a `screen` so it shouldn't be uppercase